### PR TITLE
Updated CHANGELOG.md for new release

### PR DESCRIPTION
**Pull request by scshitole**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,6 @@ install:
 - make tools
 
 script:
-- make lint
```
Comment (dannyk81) [inline .travis.yml]: This will make Travis happy 😄 but won't make all these lint issues go away...

**Comment by dannyk81** [comment]
@scshitole lookslike `make test` fails too

```
==> Checking that code complies with gofmt requirements...
gofmt needs running on the following files:
./bigip/resource_bigip_sys_snmp_traps.go
./bigip/resource_bigip_ltm_virtual_server.go
You can use the command: `make fmt` to reformat code.
make: *** [fmtcheck] Error 1
The command "make test" exited with 2.
```

need to run `go fmt` on all files I suppose.

File: .travis.yml (hunk 1 above)
Comment (scshitole) [reply .travis.yml]: @dannyk81  currently its failing for gofmt, I did run go fmt on these two files not sure why its failing again on those files, due to this its not building the images, I ran all tests passed
https://travis-ci.org/terraform-providers/terraform-provider-bigip/builds/475056797?utm_source=github_status&utm_medium=notification

**Comment by scshitole** [comment]
@dannyk81  I did run go fmt on all files, Let me close this PR and will open a fresh PR

**Comment by dannyk81** [comment]
I just ran `go fmt` on master:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   resource_bigip_ltm_virtual_server.go
        modified:   resource_bigip_sys_snmp_traps.go
```

`resource_bigip_ltm_virtual_server.go` and `resource_bigip_sys_snmp_traps.go` needs formatting.

**Pull request closed without merging**